dnsimple/dnsimple_test.go

```diff
@@ -32,6 +32,7 @@ func teardownMockServer() {
 	server.Close()
 }
 
+// TODO(@appops) Remove auxiliary functions
```

Comment:
Please do not merge into main with a TODO. Make sure this is either cleared or removed, before merging.